client/js/lounge.js

```diff
@@ -130,7 +131,8 @@ $(function() {
 			const args = text.substr(1).split(" ");
 			const cmd = args.shift().toLowerCase();
 
-			if (typeof utils.inputCommands[cmd] === "function" && utils.inputCommands[cmd](args)) {
+			if (commands[cmd]) {
+				commands[cmd].input(...args);
```

Comment:
You changed the logic, as `return false` only happened if the command itself returned `true`.

Because we want `/join` command to be sent even if it's handled.

Reply:
so 

```
			if (commands[cmd]) {
                             if (commands[cmd].input(...args)) {
				return false;
      			     }
			}
```

?

Reply:
So, we never want it to `return false`? As we always want the command to be sent?

Reply:
Well with how the commands work right now, we don't want to send collapse/expand commands to the server (as they're handled and sent separately).

Reply:
Fixed